.github/workflows/darts-cifar10-e2e-test.yaml

```diff
@@ -2,6 +2,9 @@ name: E2E Test with darts-cnn-cifar10
 on:
   - pull_request
 
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why do we need GITHUB_TOKEN? I don't think, it is exposed to pull_request type for forked repos anways.

Reply:
@johnugeorge Thanks for your review.
According to [this](https://github.com/manusa/actions-setup-minikube#optional-input-parameters) document, it seems that [actions-setup-minikube](https://github.com/manusa/actions-setup-minikube) calls GitHub REST API.
So, I added GITHUB_TOKEN.

Would you like to remove GITHUB_TOKEN?

Reply:
For unauthenticated requests, it is limited to 60 per hour https://docs.github.com/en/rest/overview/resources-in-the-rest-api#requests-from-personal-accounts Hence, we will need GitHub token if we use this action.

Reply:
Yes, it makes sense.
I don't have any concerns about the security issues of GITHUB_TOKEN. Do you have any concerns?